swagger.go

```diff
@@ -88,7 +88,7 @@ func CustomWrapHandler(config *Config, h *webdav.Handler) gin.HandlerFunc {
 				URL:         config.URL,
 				DeepLinking: config.DeepLinking,
 			})
-		case "doc.json":
+		case config.URL:
```

Comment:
I don't see any test case proving this change doesn't break anything. 
I'm asking myself if you understand how this handler is working.

Reply:
This change is unsafely, the regex matcher hardcodes the `"doc.json"` in L56.
https://github.com/swaggo/gin-swagger/blob/82c686cbb2d187ccc8b79c23020bd1cdd0d5b3b6/swagger.go#L56
If we change `"doc.json"` to `config.URL`  and `config.URL` is not `"doc.json"`, we will get a 404 error.
https://github.com/swaggo/gin-swagger/blob/82c686cbb2d187ccc8b79c23020bd1cdd0d5b3b6/swagger.go#L66-L70